.github/workflows/reusable_workflow_test.yml

```
@@ -40,6 +40,7 @@ jobs:
         with:
           flags: unittests
           files: .nyc_output/coverage-final.json
+          fail_ci_if_error: true
```

```suggestion
          fail-ci-if-error: true
```

заметил, что многие экшены используют `kebab-case` для нейминга, было бы неплохо и нам придерживаться такого